package/abs/rancher-abs

```diff
@@ -0,0 +1,464 @@
+#!/bin/bash
+if [ -e "$(dirname $0)/common.sh" ]; then
+    source $(dirname $0)/common.sh
+elif [ -e "$(dirname $0)/../common/common.sh" ]; then
+    source $(dirname $0)/../common/common.sh
+fi
+
+readonly ACCESS_KEY_ID=${ECS_ACCESS_KEY_ID}
+readonly ACCESS_KEY_SECRET=${ECS_ACCESS_KEY_SECRET}
+
+readonly LOOP_CHECK_TIMES=30
+readonly WAIT_SLEEP_TIME_IN_SECONDS=1
+
+TRANSCATION_SEQ=""
+INSTANCE_ID=""
+DISK_ID=""
+
+
+LOG() {
```

Comment:
Don't use upper case function names. You can refer to https://google.github.io/styleguide/shell.xml